### DISCUSSION TITLE
Bit of a long shot, but... pressed memtest86+ DVDs?

### DISCUSSION THREAD

**Discussion by tromlet** [memtest86plus/memtest86plus#275] [Q&A]
Hey all -

This is a bit of a long shot, but is there such a thing as a pressed memtest86+ DVD? I was thinking that that might be something useful to have for people who still have really old systems in production, since some of these systems cannot boot off of USB, and burned optical disks tend to lose bits over time with the light-sensitive organic data layer's exposure to light and the elements. DVD is, of course, preferable to CD, since it has those *two* protective 0.6mm layers of plastic sandwiching the data layer.

EDIT: lol, as I say this regarding an old Pentium II-based server that lacks a DVD drive... :P

**Comment by laozhoubuluo** [comment]
Considering the current update cycle every few months, it is believed that the new version of CD/DVD can be burned before the old version of CD/DVD becomes invalid due to light-sensitive organic data layer's exposure to light and the elements.
